kqueen_ui/blueprints/ui/views.py

```diff
@@ -749,6 +749,19 @@ class ClusterDetail(KQueenView):
     validation_hint = 'uuid'
 
     def handle(self, cluster_id):
+        def _update_status_with_floatingips(status, cluster):
+            if 'resources' not in cluster['metadata']:
+                return status
+            instances = (*cluster['metadata']['resources']['masters'], *cluster['metadata']['resources']['slaves'])
```

Comment:
I suggest defining a new variable for `cluster['metadata']['resources']` to make this more readable.